### ISSUE TITLE
[IntelliJ][ReportedByUser] Uncaught Exception Unhandled exception in [StandaloneCoroutine{Cancelling}@7d3ecc77, EDT] com.intellij.openapi.vfs.InvalidVirtualFileAccessException: Accessing invalid virtual file: file:///Users/username/Desktop/featuremind/repos/ws-sfcc/sfcc/cartridges/app_custom_core_ext; original:45622; found:-; File.exists()=false

### ISSUE THREAD

**Issue by iddemirjs** [smokeelow/intellij-sfcc#15]
IntelliJ build version: 2022.3 WS-223.7571.168
OS: Mac OS X
JDK: JetBrains s.r.o. 17.0.5
Plugin version: 2022.3
Additional Info: None
Parent component: 
```
com.intellij.openapi.wm.impl.IdeRootPane[,0,0,1792x1009,layout=javax.swing.JRootPane$RootLayout,alignmentX=0.0,alignmentY=0.0,border=com.intellij.ui.ToolbarUtil$3@15fe94c1,flags=449,maximumSize=,minimumSize=,preferredSize=]
```
Error message: 
```
IdeaLoggingEvent[message=Unhandled exception in [StandaloneCoroutine{Cancelling}@7d3ecc77, EDT], throwable=com.intellij.openapi.vfs.InvalidVirtualFileAccessException: Accessing invalid virtual file: file:///Users/idrisdemir/Desktop/featuremind/repos/ws-sfcc/sfcc/cartridges/app_custom_core_ext; original:45622; found:-; File.exists()=false
	at com.intellij.openapi.vfs.newvfs.impl.VirtualDirectoryImpl.handleInvalidDirectory(VirtualDirectoryImpl.java:183)
	at com.intellij.openapi.vfs.newvfs.impl.VirtualDirectoryImpl.doFindChild(VirtualDirectoryImpl.java:108)
	at com.i

**Comment by smokeelow** [comment]
Hi, when this issue happened ? On branch change?

**Comment by iddemirjs** [comment]
Yes It is coming when I changed my branch

**Comment by smokeelow** [comment]
got it, reproduced.
will fix in next release

**Comment by smokeelow** [comment]
Fixed in `2022.3.5`
